docs/search_algorithm_switching.md

```diff
@@ -1,37 +1,533 @@
 # Search Algorithm Switching
 
-As part of the Search and Race project work, we've decided to explore allowing users to
-submit their searches using different search algorithms.
+As part of the Search and Race project work, we've decided to explore allowing users to execute their catalog search queries using alternative algorithms used for querying Apache Solr as well as for ranking the search results.
 
-## Context
+## Technical Implementation
 
-We could do this primarily in ruby code or primarily in Solr.
+### Apache Solr Request Handler
 
-### Solr Request Handler
+_(Please refer to [the documentation](https://solr.apache.org/guide/solr/latest/configuration-guide/requesthandlers-searchcomponents.html#RequestHandlersandSearchComponentsinSolrConfig-RequestHandlers) for this component of Apache Solr.)_
```

Comment:
Can we rename the href text to: 'Request handler search component documentation'